pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -173,6 +173,20 @@ var _ = Describe("Clone", func() {
 		})
 	}
 
+	expectRestoreCreationFailure := func(snapshotName string, vmClone *clonev1alpha1.VirtualMachineClone, restoreName string) {
```

Comment:
Need to avoid this repetition, could wrap it in a function that receives the error message and returns this function.